src/App.js

```diff
@@ -48,13 +67,19 @@ class App extends Component {
   async handleUrl(item){
     try {
 
-      const url = `${starWarsApiUrl}?search=${item}`;
+      const url = `${Constants.STARWARS_API_URL}?search=${item}`;
       const response = await fetch(url);
       const json = await response.json();
       
-      // TODO: change logic here
       if (json.count >= 1) {
-        return Promise.resolve(json.results[0].name);
+        const resultObject={
```

Comment:
You can destructure this, for instance 
```js
const { name, height, mass, birth_year, gender } = json.results[0]
```
and then just create the object like so:
```js
const resultObject = {
 name,
 height, 
 mass,
 birth_year, 
 gender
}
```